src/js/itemset.js

```diff
@@ -198,11 +198,11 @@ export default {
                                     // Two falsy values should set active to true.
                                     const active = ( !lang && !that.form.langs.currentLanguage ) || ( lang === that.form.langs.currentLanguage );
 
-                                    return { language: lang, type: 'span', text: label.textContent, active };
+                                    return { language: lang, type: 'span', contentNodes: [ ...label.childNodes ], active };
                                 } );
                                 break;
                             default:
-                                translations = [ { language: '', type: 'span', text: labels && labels.length ? labels[ 0 ].textContent : 'error', active: true } ];
+                                translations = [ { language: '', type: 'span', contentNodes: labels && labels.length ? [ ...labels[ 0 ].childNodes ] : [], active: true } ];
```

Comment:
This previously populated `'error'` when there were no labels, now appears to populate nothing. Is that intentional? I'm not sure what's expected here.

Reply:
It's intentional. I think it makes more sense to show an empty label if those nodes are missing or falsy.